mock-core-configs/etc/mock/chroot-aliases.cfg

```diff
@@ -0,0 +1,32 @@
+config_opts["no-config"]["epel-8"] = {
+    "/etc/mock/alma+epel-8": {
+        "description": [
+            "Builds against AlmaLinux repositories, "
+            "together with the official EPEL repositories.",
+            "Project page: https://almalinux.org/"
```

Comment:
I would love to see here command that will resolve it. E.g.:

    ln -s alma+epel-8-{{arch}} /etc/mock/epel-8-{{arch}}

So users can copy'n'paste it directly

Reply:
For N alternatives, N-times `ln -s` command in the output?

Reply:
Fixed.